packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/container.ts

```diff
@@ -51,6 +61,11 @@ export class Container extends ServiceExtension {
    */
   public readonly trafficPort: number;
 
+  /**
+   * The default log group into which application container logs should be routed.
```

Comment:
By default it means that if the user doesn't provide a log group, logging will still be enabled as default behavior for the service. But there won't be multiple log groups for an application container so I can see where it might get confusing.